packages/web-features-figuring/src/browser-compat-data/typeUtils.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  Browsers,
+  BrowserStatement,
+  CompatData,
+  CompatStatement,
+  Identifier,
+  MetaBlock,
+} from "@mdn/browser-compat-data";
+
+export function isIndexable(o: unknown): o is Record<string, unknown> {
+  if (typeof o === "object" && o !== null && Object.keys(o).length > 0) {
```

Comment:
Why is the `Object.keys(o).length` check needed here? Shouldn't `{}` be considered indexable even though it has no keys?

I thought this might be to handle `Map`, but `Object.keys()` for a map returns an empty array.

Reply:
Good catch. Removed the length check to no ill effect.